docs/auth/google/gcp-iap-auth.md

```diff
@@ -94,6 +101,27 @@ Now the backend is ready to serve auth requests on the
 `/api/auth/gcp-iap/refresh` endpoint. All that's left is to update the frontend
 sign-in mechanism to poll that endpoint through the IAP, on the user's behalf.
 
+### New Backend System
```

Comment:
Would this be a good time to restructure the page to include only the "new" way of installing the plugin and module and create an "old" page with the content for people who are using the legacy backend system?

Reply:
After thinking this over I think in this case as it's just a small section it doesn't make sense to have a full "old" page. what I did instead was invert this so the new backend system section is the default and the old backend system is below that.